tests/test_stategraph.py

```diff
@@ -0,0 +1,85 @@
+
+import os
+
+import claripy
+import angr
+from angr.analyses.state_graph_recovery import MinDelayBaseRule, RuleVerifier, IllegalNodeBaseRule
+
+
+binaries_base = os.path.join(os.path.dirname(os.path.realpath(__file__)), '..', '..', 'binaries')
+
+
+def test_smoketest():
+    path = r"C:\Users\Fish\Desktop\temp\mitre\Traffic_Light_Short_Ped\build\Traffic_Light_Short_Ped.so"
```

Comment:
The path seems not in the `binaries` directory ???

Reply:
This branch will not be merged to master any time soon. I will make sure to move this binary to the binaries repo before merging to master. Thanks for pointing out!